oeps/oep-0002.rst

```diff
@@ -69,6 +70,32 @@ following fields:
     If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
     ``False``, unless the reporting tool can auto-detect accordance.
 
+``track-pulls``: boolean (optional)
+
+    .. This should be moved to a separate OEP
+
+    Whether or not this repository should be tracked for creation of OSPR tickets.
```

Comment:
There's an irony to having this key any more.  When read from repos.yaml, it meant that we didn't have to look at every repo in the organization. Now that it's distributed, we have to look at every repo to see if this is set. Probably we should just check every repo no matter what.  This was just an optimization.